pretty_midi/pretty_midi.py

```diff
@@ -1258,6 +1353,180 @@ def __repr__(self):
                 'time={:f})'.format(self.number, self.value, self.time))
 
 
+class TimeSignature(object):
+    """Containts the time signature numerator, denominator and
+    the event time in seconds
+
+    Attributes
+    ----------
+        numerator : int
+            Numerator of time signature
+        denominator : int
+            Denominator of time signature
+        time : float
+            Time of event in seconds
+
+    Examples
+    --------
+    Instantiate a TimeSignature object with 6/8 time signature at 3.14 seconds
+    >>> ts = TimeSignature(6, 8, 3.14)
+    >>> print ts
+    6/8 at 3.14 seconds
+
+    """
+
+    def __init__(self, numerator, denominator, time):
+        assert isinstance(numerator, int), \
+            '%s is not a recognized `numerator`' % str(type(numerator))
+        assert isinstance(denominator, int), \
+            '%s is not a recognized `denominator`' % str(type(denominator))
+        assert isinstance(time, float), \
+            '%s is not a recognized `time`' % str(type(key_number))
+
+        self.numerator = numerator
+        self.denominator = denominator
+        self.time = time
+
+    def __repr__(self):
+        return '%d/%d at %.2f seconds' % (
```

Comment:
`__repr__` is meant to produce code which can re-create the object (see `Note`) for example, so this should be changed to something like 
`"TimeSignature(numerator={}, denominator={}, time={})".format(self.numerator, self.denominator, self.time)`